powerdns_recursor/datadog_checks/powerdns_recursor/powerdns_recursor.py

```diff
@@ -126,6 +126,27 @@ def check(self, instance):
                     self.gauge('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
                 elif stat['name'] in PowerDNSRecursorCheck.RATE_METRICS_V4:
                     self.rate('powerdns.recursor.{}'.format(stat['name']), float(stat['value']), tags=tags)
+        self._collect_metadata(config)
+
+    def _collect_metadata(self, config):
+        url_v4 = "http://{}:{}/api".format(config.host, config.port)
+        url = "http://{}:{}/servers/localhost/statistics".format(config.host, config.port)
+
+        try:
+            response = self._get_pdns_response(config, url, url_v4)
+        except Exception as e:
+            self.log.warning('Error collecting PowerDNS Recursor version: %s', str(e))
```

Comment:
Warning logs have proven to be too noisy and generate support cases as customers wonder if there's anything super wrong when they see them (see #6930), IMO we should use `DEBUG`:

```suggestion
            self.log.debug('Error collecting PowerDNS Recursor version: %s', str(e))
```